src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -106,6 +106,22 @@ export const QueryResultPane = () => {
             renderHeaderCell: () => <>{MESSAGE}</>,
         }),
     ];
+	const gridParentRef = useRef<HTMLDivElement>(null);
```

Comment:
Fix formatting. It's not in line with the previous one.

Reply:
That's weird, it doesn't look like that on my machine

Reply:
Check that indentation is 4 spaces and not a tab. That's usually what causes it.

Reply:
@cssuh , maybe merge from main and format the file.